src/scvi/external/mrvi/_module.py

```diff
@@ -202,7 +286,55 @@ def __call__(
 
 @flax_configure
 class MrVAE(JaxBaseModuleClass):
-    """Flax module for the Multi-resolution Variational Inference (MrVI) model."""
+    """Multi-resolution Variational Inference (MrVI) module.
+
+    Parameters
+    ----------
+    n_input
+        Number of input features.
+    n_sample
+        Number of samples.
+    n_batch
+        Number of batches.
+    n_labels
+        Number of labels.
+    n_continuous_cov
+        Number of continuous covariates.
+    n_latent
+        Number of latent variables.
+    n_latent_u
+        Number of latent variables for ``u``.
+    encoder_n_hidden
+        Number of hidden units in the encoder.
+    encoder_n_layers
+        Number of layers in the encoder.
+    z_u_prior
+        TODO
+    z_u_prior_scale
+        TODO
+    u_prior_scale
+        TODO
+    u_prior_mixture
+        TODO
+    u_prior_mixture_k
+        TODO
+    learn_z_u_prior_scale
+        TODO
+    laplace_scale
+        TODO
+    scale_observations
+        TODO
```

Comment:
Would love to get input on these

Reply:
z_u_prior
    Whether to place a Gaussian prior on ``z`` given ``u``.
z_u_prior_scale
    Natural log of the scale parameter of the Gaussian prior placed on ``z`` given ``u``. Only applies of ``learn_z_u_prior_scale=False``.
u_prior_scale
    Natural log of the scale parameter of the Gaussian prior placed on ``u``. If ``u_prior_mixture=True``, this scale applies to each mixture component distribution.
u_prior_mixture
    Whether to use a mixture of Gaussians prior for ``u``.
u_prior_mixture_k
    Number of mixture components to use for the mixture of Gaussians prior on ``u``.
learn_z_u_prior_scale
    Whether to learn the scale parameter of the prior distribution of ``z`` given ``u``.
scale_observations
    Whether to scale the loss associated with each observation by the total number of observations linked to the associated sample.
    

Let's remove laplace scale (it was used for the old linear model) and all traces of "As"